src/main/java/com/logentries/net/LogentriesClient.java

```diff
@@ -227,23 +227,8 @@ private SSLContext CreateAndInitSSLContext(String algorithm) throws NoSuchAlgori
 			KeyManagementException {
 
 		SSLContext sslContext = SSLContext.getInstance(algorithm);

```

Comment:
I think it does not make much sense to use a custom context if we do not use it. 
The idea would be to get the SSL behaviour correct